src/test/scala/org/broadinstitute/dsde/rawls/expressions/RhinoTest.scala

```diff
@@ -0,0 +1,707 @@
+package org.broadinstitute.dsde.rawls.expressions
+
+import javax.script.{Invocable, ScriptEngine, ScriptEngineManager}
+
+import com.tinkerpop.blueprints.{Direction, Edge, Vertex, Graph}
+import com.tinkerpop.gremlin.java.GremlinPipeline
+import com.tinkerpop.pipes.PipeFunction
+import org.broadinstitute.dsde.rawls.graph.OrientDbTestFixture
+import org.scalatest.FunSuite
+import org.mozilla.javascript._
+import scala.collection.JavaConversions._
+import scala.util.Random
+import scala.util.parsing.combinator.JavaTokenParsers
+
+/**
+ * Created by abaumann on 5/21/15.
+ */
+class RhinoTest extends FunSuite with OrientDbTestFixture {
+  override val testDbName = "ParserCombinatorTest"
+
+  def initializeGraph: Unit = {
+    val workspace = graph.addVertex(null, "_namespace", "workspaces", "_name", "Workspace1", "_entityType", "Workspace")
+
+    val sample1 = graph.addVertex(null, "_name", "sample1", "_entityType", "Sample", "type", "normal")
+    val sample2 = graph.addVertex(null, "_name", "sample2", "_entityType", "Sample", "type", "tumor")
+    val sample3 = graph.addVertex(null, "_name", "sample3", "_entityType", "Sample", "type", "tumor")
+    val pair1 = graph.addVertex(null, "_name", "pair1", "_entityType", "Pair")
+    val pair2 = graph.addVertex(null, "_name", "pair2", "_entityType", "Pair")
+    val sampleSets = graph.addVertex(null, "_name", "sampleSets")
+    val sampleSet1 = graph.addVertex(null, "_name", "sampleSet1", "_entityType", "SampleSet")
+    val sampleSet2 = graph.addVertex(null, "_name", "sampleSet2", "_entityType", "SampleSet")
+    val pairSet1 = graph.addVertex(null, "_name", "pairSet1", "_entityType", "Pair")
```

Comment:
Dunno if this is already a TODO, but we should put reserved keywords like ```_name``` and ```_entityType``` in a single object somewhere.